src/main/java/com/zotov/edu/passportofficerestservice/repository/converter/PersonEntityConverter.java

```diff
@@ -0,0 +1,27 @@
+package com.zotov.edu.passportofficerestservice.repository.converter;
+
+import com.zotov.edu.passportofficerestservice.repository.entity.Person;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+@Component
+public class PersonEntityConverter {
+
+    public Person convertToEntity(String name, LocalDate birthday, String country) {
+        return new Person(
```

Comment:
both `convertToEntity` have much in common. Better to call one from another to avoid duplication

Reply:
The method has been refactored.